server/server.js

```diff
@@ -1,15 +1,36 @@
 var express = require('express');
+var expressLogging = require('express-logging');
+var logger = require('logops');
 var path = require('path');
-var gameHandler = require('./handlers/game_handler');
+var Cookies = require('universal-cookie');
+var firebaseRequests = require('./firebase.js');
+var gameHandler = require('./handlers/game_handler.js');
+
 var app = express();
 
 app.use("/static", express.static(path.join(__dirname, "/../public")))
 
+app.use(expressLogging(logger));
 app.get("/", function(req, res, next) {
     var filePath = path.join(__dirname + "/../index.html");
     res.sendFile(filePath);
 });
 
+app.use(function(req, res, next) {
+  const cookies = new Cookies(req.headers.cookie);
+  console.log(req.headers);
+  var idToken = cookies.get('JWT');
+  console.log(idToken);
+  if(idToken) {
+    if(firebaseRequests.verifyIdToken(idToken)) {
```

Comment:
does this actually return any info about the user? like an id or something? i think for any request we'd want to know who is the user, so if this verify request passes then we'd want to store an id or some info about the verified user so that the endpoint could use it (i.e. if they hit /api/game/create, we need to know who is the actual user creating the game)

Reply:
yeah i think it took this out before committing but in the verifyIdToken method you eventually receive a decoded token back which i was logging to console server-side for testing--it has lots of identifying info in it